docs/source/includes/elasticluster_setup.rst

```diff
@@ -0,0 +1,358 @@
+.. _S3IT: http://www.s3it.uzh.ch/
+.. _Elasticluster: https://elasticluster.readthedocs.org
+.. _virtualenv: http://docs.python-guide.org/en/latest/dev/virtualenvs/
+.. _gcloud: https://cloud.google.com/sdk/
```

Comment:
Consider re-using or adding to the common links here: https://github.com/googlegenomics/start-here/blob/master/docs/source/conf.py#L139)